### ISSUE TITLE
Allow user to take experiment tour at any time

### ISSUE THREAD

**Issue by sevaan** [mozilla/testpilot#1483]
Currently experiment tours can only be view when a user enables an experiment. To see the tour again, the user must disable the experiment and then re-enable it.

Users should be able to access this tour at any time.

**Comment by sevaan** [comment]
Suggestion:

In the experiment details, next to the Version number, we have a link to the changelog. Let's add a comma and a link to the tour next to it. Clicking the link launches the tour.

<img width="360" alt="tour" src="https://cloud.githubusercontent.com/assets/473531/18997994/c76f34da-8704-11e6-8ac5-8a9e91c2b482.png">

**Comment by lmorchard** [comment]
One caveat to the UX for this issue: The block with changelog & etc only shows up when you have the main Test Pilot add-on installed. So, in PR #1567, this tour link will not appear for users without the add-on

**Comment by SoftVision-PaulOiegas** [comment]
@sevaan We did some exploratory testing after the link was added, and just as a personal opinion, we feel that the link for the tour is not quite visible for the end user. It's very small and it's in a section that is more related to technical details about the experiment, so it will be easy to pass unnoticed.

Wouldn't be better to display a more visible button for it somewhere at the top between description and screenshots of the experiment, or even on the bottom of the page under all the screenshots ? 
I think the user would probably like to see a tour of the experiment after he finds out some details about it.
